tests/system/src/test/java/test/util/Util.java

```diff
@@ -466,4 +471,28 @@ public static boolean isFractionalScaleX(Node n) {
     private static boolean isFractional(double x) {
         return x != Math.rint(x);
     }
+
+    /**
+     * Returns true if the operating system is a form of Windows.
+     * @return true if the platform is Windows
+     */
+    public static boolean isWindows(){
+        return WINDOWS;
```

Comment:
Are these really needed or can the tests call PlatformUtil directly? If there is a good reason to add them, then they should delegate to PlatformUtil rather than duplicate the functionality.

Reply:
was able to use PlatformUtils directly after adding 
--add-exports javafx.base/com.sun.javafx=ALL-UNNAMED
to the launch configuration.